python/cuml/fil/fil.pyx

```diff
@@ -258,7 +265,8 @@ cdef class ForestInference_impl():
                 <size_t> n_rows,
                 <bool> predict_proba)
         self.handle.sync()
-        # synchronous w/o a stream
+        del(X_m)
```

Comment:
Why do we need these two dels? they should go out of scope anyways at the end of the function and be fine.
Del just removes the local name binding of X_m -> underlying data. So it should hardly ever be necessary in a function we're about to return from

Reply:
sorry, had them in while trying to figure out why the tests were getting Killed and forgot to remove it